vendor/src/github.com/docker/engine-api/types/types.go

```diff
@@ -274,6 +274,19 @@ type ExecStartCheck struct {
 	Tty bool
 }
 
+type HealthcheckResult struct {
+	Start, End time.Time
+	ExitCode   int    // 0=healthy, 1=unhealthy, 2=starting
+	Output     string // Output from last check
```

Comment:
I can't make any guarantees about all json libraries, but the json grammar is very unambiguous that all unicode characters except `"`, `\`, and control characters are valid in strings. Go will escape those exceptions.